public/js/leaderboard.js

```diff
@@ -2,18 +2,51 @@
 $('#nav-leaderboard').addClass('active');
```

Comment:
why cant we add this in the pug directly

Reply:
I didn't add this so not sure :laughing: